themes/digital.gov/layouts/partials/core/topics-button.html

```diff
@@ -0,0 +1,50 @@
+{{/* Renders a topics button with the name of the topic and has two display options.
+  Option 1 is a styled button with a hover effect and Option 2 is a link followed by the amount of pages that use the topic.
+
+  @param {url} link - Accepts a string path to a topic page
+  @param {string} title - Name of the topic
+  @param {number} tag_count - Number of times the topic tag is used
+  @param {boolean} has_tag_count - Checks if tag_count number is present
+*/}}
```

Comment:
Updates for readability. Please use full sentences.

```suggestion
{{/* Renders a topics button with the name of the topic and has two display options.

  1. Primary - Styled button with a hover effect.
  2. Count - Link followed by the amount of pages that use the topic.

  @param {url} link - String path to a topic page.
  @param {string} title - Topic name.
  @param {number} tag_count - Number of times the topic tag is used.
  @param {boolean} has_tag_count - Checks if `tag_count` number is present.
*/}}
```

Reply:
thank you for pointing that out.. updated to use full sentences